src/frontend/handlers/__snapshots__/home.test.ts.snap

```diff
@@ -0,0 +1,226 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`home page handler Page displays all overview rows 1`] = `
+"<!DOCTYPE html>
+<html lang="en" class="govuk-template ">
+  <head>
+    <meta charset="utf-8">
+    <title>
+  
+  Billings and reconciliation -
+  Billings and Reconciliation
+</title>
```

Comment:
Yah, it is weird and also weird that it's duplicated (as it's adding on to the base) but maybe one for James and/or UX to decide if a change is needed as it will impact all the pages.